packages/react-components/style/trace-explorer.css

```diff
@@ -74,16 +74,12 @@
 }
 
 .trace-element-name, .outputs-element-name {
-    overflow: hidden;
-    text-overflow: ellipsis;
```

Comment:
These changes seem related and necessary to the first commit, but the first commit is unrelated to this PR.

If you intended to make these changes in a separate PR, the first commit and this commit should simply be removed from this PR.